src/content/docs/en/install/auto.mdx

```diff
@@ -85,6 +85,8 @@ If all goes well, Astro should now be serving your project on [http://localhost:
 
 Astro will listen for live file changes in your `src/` directory, so you will not need to restart the server as you make changes during development.
 
+When viewing your site in the browser, you'll have access to the [Astro dev toolbar](/en/reference/configuration-reference/#dev-toolbar-options). As you build, it will help you inspect your [islands](/en/concepts/islands/), spot accessibility issues, and more.
```

Comment:
Nitpicking: It already says "Astro _dev_ toolbar", but wouldn't it be helpful to say that this only happens in DEV mode?

Reply:
I actually started with that included, but there is a whole lot of "dev" in this section already, so it felt a bit much in context!  😄